docs/manual-test.md

```diff
@@ -179,6 +179,11 @@ The workbook should publish without errors.
         1. In the Publish Data Source dialog box, under **Authentication**, click __Edit__ next to **Refresh not enabled**.
         1. Under **Authentication**, select **Allow refresh access** from the drop-down list.<br/>
         ![]({{ site.baseurl }}/assets/mt-pub-allow-refresh.png)<br/>
+        1. For a datasource using OAuth, go to the server settings page to add your credential for the datasource.
+        ![]({{ site.baseurl }}/assets/oauth-server-addtoken.png)<br/> 
+        1. Then under **Authentication** select **Embed <username>** from the drop-down list.
+        ![]({{ site.baseurl }}/assets/oauth-desktop-publish.png)<br/> 
+
```

Comment:
Can you separate out the OAuth steps into their own section? So we'd have "General Manual Tests" and "OAuth Manual Tests". That way, when we get the screenshots/video of the results it's easier to find specifically the OAuth test cases.

Reply:
Yeah sure I can do that.